bin/desi_eval_prod_jobs

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python
+
+"""
+Get queue info for production jobs and summarize
+"""
+
+import os, sys, glob, argparse
+import numpy as np
+from astropy.table import Table, vstack
+
+import desispec.io
+from desispec.workflow.tableio import load_table
+from desispec.workflow.queue import queue_info_from_qids
+
+def load_proctables(specprod=None):
+    """
+    Return list of processing tables, loaded from a production
+
+    Options:
+        specprod (str): override $SPECPROD, production name
+
+    Returns: list of proctables
+    """
+
+    proddir = desispec.io.specprod_root(specprod=specprod, readonly=True)
+    procfiles = sorted(glob.glob(f'{proddir}/processing_tables/processing_table_{specprod}-????????.csv'))
+
+    proctables = [load_table(fn, tabletype='proctable', suppress_logging=True) for fn in procfiles]
+    return proctables
+
+def hhmmss2hours(hhmmss):
+    """
+    Convert an hh:mm:ss string into floating point hours
+    """
+    hh, mm, ss = hhmmss.split(':')
+    hours = int(hh) + int(mm)/60. + int(ss)/3600.
+    return hours
+
+def get_zpix_qids(specprod=None):
+    """
+    Return list of zpix jobids, parsed from log filenames
+
+    Options:
+        specprod (str): override $SPECPROD, production name
+
+    Returns: list of jobids
+    """
+    proddir = desispec.io.specprod_root(specprod)
+    zpixlogs = sorted(glob.glob(f'{proddir}/run/scripts/healpix/*/*/*/zpix-*.log'))
+    zpix_qids = list()
+    for filename in zpixlogs:
+        qid = os.path.splitext(os.path.basename(filename))[0].split('-')[-1]
+        zpix_qids.append(qid)
+
+    return zpix_qids
+
+def load_qinfo(specprod=None):
+    """
+    Load queue job info for a spectroscopic production
+
+    Options:
+        specprod (str): override $SPECPROD, production name
+
+    Returns: qinfo Table with columns JOBID,JOBNAME,PARTITION,CONSTRAINTS,NNODES,SUBMIT,ELIGIBLE,START,END,ELAPSED,STATE,EXITCODE
+    """
+
+    ptables = load_proctables(specprod)
```

Comment:
An optimization suggestion you don't have to take if you don't want: If you vstack these, then you could avoid a for loop over rows and instead loop over jobdesc

```
jobdesc_qids = dict()
for ptab in ptables:
      for jobdesc in np.unique(ptab['JOBDESC']):
            qids = list(np.concatenate(ptab['ALL_QIDS'][ptab['JOBDESC']==jobdesc]))
            if jobdesc not in jobdesc_qids:
                  jobdesc_qids[jobdesc] = qids
            else:
                  jobdesc_qids[jobdesc].extend(qids)
```

Originally I thought doing vstack and avoid the loop over ptables would be faster, but it turns out it isn't. But the above is ~25% faster than looping over individual rows.